src/backend/api/Fusion.Resources.Domain/Services/ProfileServices.cs

```diff
@@ -174,7 +191,7 @@ public async Task<DbPerson> EnsureSystemAccountAsync()
                 person = new DbPerson
                 {
                     AccountType = $"{FusionAccountType.Application}",
-                    AzureUniqueId = Guid.Empty,
+                    AzureUniqueId = azureUniqueId,
```

Comment:
This was changed to Guid.Empty earlier this year, but cannot really understand why it was changed